x/ibc/02-client/exported/exported.go

```diff
@@ -135,9 +134,13 @@ type ConsensusState interface {
 
 // Misbehaviour defines counterparty misbehaviour for a specific consensus type
 type Misbehaviour interface {
-	evidenceexported.Evidence
 	ClientType() ClientType
 	GetClientID() string
+	String() string
+	ValidateBasic() error
+
+	// Height at which the infraction occurred
+	GetHeight() uint64
```

Comment:
readded relevant methods from `Evidence` interface. `GetHeight` will be changed to return abstract height in the upcoming abstract height PR